openlcsd/flow/tasks.py

```diff
@@ -829,8 +831,8 @@ def get_remote_source_components(context, engine):
     """
     Get remote source components in source container
     @requires: `config`, configurations from Hub.
-    @requires: `package_nvr`, nvr of the container.
-    @feeds: `rs_components`, remote source components in source container.
+    @requires: `binary_build`, brew build for the binary NVR.
+    @returns: `rs_components`, remote source components in source container.
```

Comment:
In the tasks.py, Looks like we use `feeds`(I don't know why use this one), so if we update, I think need to update all of them in this file